globus_automate_client/flows_client.py

```diff
@@ -491,7 +492,7 @@ def list_flows(
             filters.pop("per_page", None)
             params.update(filters)
 
-        return self.get("/flows", params=params, **kwargs)
+        return self.get("/flows", query_params=params, **kwargs)
```

Comment:
Is `params` -> `query_params` an SDK3 change? I wouldn't have expected that change as it doesn't seem to add much (and maybe changes from what `requests` does which would have been a reasonable thing to follow?)

Reply:
This changed throughout the SDK's request methods. See [globus_sdk.client.BaseClient.get()](https://github.com/globus/globus-sdk-python/blob/main/src/globus_sdk/client.py#L120).